src/XMakeCommandLine/Microsoft.Build.CommonTypes.xsd

```diff
@@ -168,6 +168,51 @@ elementFormDefault="qualified">
       </xs:complexContent>
     </xs:complexType>
   </xs:element>
+  <xs:element name="PackageReference" substitutionGroup="msb:Item">
+    <xs:annotation>
+      <xs:documentation>
+        <!-- _locID_text="PackageReference" _locComment="" -->Reference to a package
+      </xs:documentation>
+    </xs:annotation>
+    <xs:complexType>
+      <xs:complexContent>
+        <xs:extension base="msb:SimpleItemType">
+          <xs:sequence minOccurs="0" maxOccurs="unbounded">
+            <xs:choice>
+              <xs:element name="Version">
```

Comment:
All the examples on the public docs were using it as a tag. If the goal is to deprecate the tag and move people to the attribute I can make the change to have version as an attribute and remove the tag.